edsby.py

```diff
@@ -443,15 +525,24 @@ def getAllClassAverages(self):
         Returns an object containing all assignment metadata for a specified course, ordered by RID
         This includes the NID, RID, and weighting (points possible) of the assignments, but not your score.
     """
+
     def getClassAssignmentMetadata(self, classNID):
-        return requests.get('https://'+self.edsbyHost+'/core/node.json/'+str(classNID)+'?xds=MyWork&student='+str(self.studentData['unid']),cookies=self.getCookies(),headers=self.getHeaders()).json()['slices'][0]['data']['loaddata']['gradebook']['terms']
+        return requests.get(
+            'https://' + self.edsbyHost + '/core/node.json/' + str(classNID) + '?xds=MyWork&student=' + str(
+                self.studentData['unid']), cookies=self.getCookies(), headers=self.getHeaders()).json()['slices'][0][
+            'data']['loaddata']['gradebook']['terms']
 
     """
         Returns an object containing all assignment scores for a specified course, ordered by NID
         This includes the NID and points earned on the assignment, but nothing else.
     """
+
     def getClassAssignmentScores(self, classNID, classRID):
-        return requests.get('https://'+self.edsbyHost+'/core/node.json/'+str(classNID)+'/'+str(classRID)+'/'+str(classNID)+'?xds=MyWorkAssessmentPane&unit=all&student='+str(self.studentData['unid'])+'&model=24605449',cookies=self.getCookies(),headers=self.getHeaders()).json()["slices"][0]["data"]['grades']
+        return requests.get(
+            'https://' + self.edsbyHost + '/core/node.json/' + str(classNID) + '/' + str(classRID) + '/' + str(
+                classNID) + '?xds=MyWorkAssessmentPane&unit=all&student=' + str(
+                self.studentData['unid']) + '&model=24605449', cookies=self.getCookies(),
```

Comment:
I get what you mean. What PyCharm is doing here, is it's breaking up lines that go over the PEP8 standard's 120 char per line limit. I'll look into custom formatting.